### ISSUE TITLE
Bridge Toolbar for Visual Studio

### ISSUE THREAD

**Issue by godlikesme** [Microsoft/WinObjC#219]
Any plans on including some kind of toolbar for Visual Studio to give developers a way to import XCode projects into VS via the GUI as opposed to command line?

**Comment by antoine-haas** [comment]
Yes, we are currently investigating the integration of the vsimporter tool to Visual Studio via the File -> New -> Project From Existing Code menu. We haven't set on a date yet and are organizing our dev resources.
